Directory.Packages.props

```diff
@@ -44,6 +44,6 @@
   </ItemGroup>
   <ItemGroup Condition="!$([MSBuild]::IsTargetFrameworkCompatible($(TargetFramework), 'netcoreapp3.1'))">
     <PackageVersion Include="Microsoft.Extensions.Options" Version="2.2.0" />
-    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="2.2.0" />
+    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.0" />
```

Comment:
This won't cause an issue for any of the older TFMs?

Reply:
It shouldn't and if we find so it will be fixed in 8.0.0 version that we will reference on every TFM anyway.